src/Module/Contact.php

```diff
@@ -453,9 +445,7 @@ public static function content(array $parameters = [], $update = 0)
 
 		$_SESSION['return_path'] = DI::args()->getQueryString();
 
-		if (!empty($a->data['contact']) && is_array($a->data['contact'])) {
-			$contact = $a->data['contact'];
-
+		if (!empty($contact) && is_array($contact)) {
```

Comment:
Why `is_array()`? What else can `$contact` be at this point?

Reply:
In many cases I did a blind search and replace without judging the code.